### ISSUE TITLE
Colour contrast problems on the dashboard page

### ISSUE THREAD

**Issue by jonkri** [mysociety/fixmystreet#3900]
**Describe the bug**
The colours used under All time and Last 7 days under All reports do not satisfy the WCAG colour contrast requirements (according to https://color.review/). The colors displayed on the page are `#F4A140`, `#62B356` and `#4FADED`. See also the `colours` array in `web/js/dashboard.js`. Perhaps the icons in `web/cobrands/fixmystreet.com/images/` should be updated as well.

**To Reproduce**
1. Visit All reports (/reports)
2. Notice the colour contrast between for text that is not black, and the white background colour

**Expected behavior**
The colour contrasts should satisfy the relevant requirements. Perhaps AA (4.5:1) would suffice.

**Screenshots**
![last-7-days](https://user-images.githubusercontent.com/777056/166207198-847a78d2-1595-477f-bdbf-37204c001e26.png)

**Desktop (please complete the following information):**
Irrelevant.

**Smartphone (please complete the following information):**
Irrelevant.

**Additional context**
Sufficient colour contrasts contributes to accessibility.

**Comment by jonkri** [comment]
Here are some color suggestions for satisfying the WCAG colour contrast requirements.

Using the three colors from the All reports page (`#F4A140`, `#4FADED` and `#62B356`) as a starting point, I converted the colors to HSL (for example `hsl(32 89 60)` for `#F4A140`) and then lowered the lightness (the L component) until the desired contrast threshold was reached (for example `hsl(32 89 45)` (for a contrast of 3:1) and `hsl(32 89 36)` (for a contrast of 4.5:1). Finally, I converted the HSL colors back to hex (for example `#D97B0C` (3:1) and `#AE620A` (4.5:1).

Here is the result:

| Text                     | Current   | Headlines AA (3:1) | Headlines AAA/Text AA (4.5:1) | Text AAA (7:1) |
|--------------------------|-----------|--------------------|-------------------------------|----------------|
| Problems reported        | `#F4A140` | `#D97B0C`          | `#AE620A`                     |  `#824A07`     |
| Updates on problems      | `#4FADED` | `#269AE9`          | `#1379BE`                     |  `#0F5B8F`     |
| Problems marked as fixed | `#62B356` | `#56A54A`          | `#44823B`                     |  `#33632C`     |

I'm guessing we can use the headlines (3:1) criteria for the numbers on the All reports page since the font size is 32px.

> Large text is defined as 14 point (typically 18.66px) and bold or larger, or 18 point (typically 24px) or larger.

You'll find some images below what the text looks like in the above colors. I hope this is helpful!

## Current: `#F4A140`, `#4FADED` and `#62B356`

<img width="506" alt="current" src="https://github.com/mysociety/fixmystreet/assets/777056/4800a506-04b1-4f1a-9ff8-0c34f3c08fb4">

## Headlines AA (3:1): `#D97B0C`, `#269AE9` and `#56A54A`

<img width="506" alt="headlines-aa" src="https://github.com/mysociety/fixmystreet/assets/777056/b595a771-ab3c-42af-97f7-f0faeadd1362">

## Headlines AAA/Text AA (4.5:1): `#AE620A`, `#1379BE` and `#44823B`

<img width="506" alt="headlines-aaa-text-aa" src="https://github.com/mysociety/fixmystreet/assets/777056/abede491-5a2f-493f-a1bd-7ce4e106f8c8">

## Text AAA (7:1): `#824A07`, `#0F5B8F` and `#33632C`

<img width="506" alt="text-aaa" src="https://github.com/mysociety/fixmystreet/assets/777056/b0b75a7c-551a-4dfe-8b5d-20a860c9fca5">

**Comment by dracos** [comment]
Thanks @jonkri! :)
